src/lensCalibration/LensCalibrationPlugin.cpp

```diff
@@ -227,28 +236,31 @@ void LensCalibrationPlugin::calibrateLens()
                                                                              calibInputFrames,
                                                                              calibImageScore,
                                                                              rejectInputFrames);
-  
-  setOutputParams(_outputCameraFocalLenght,
+
+  setOutputParams(_outputIsCalibrated,
+                  _outputAvgReprojErr,
+                  _outputCameraFocalLenght,
                   _outputCameraPrincipalPointOffset,
                   _outputLensDistortionRadialCoef1,
                   _outputLensDistortionRadialCoef2,
                   _outputLensDistortionRadialCoef3,
                   _outputLensDistortionTangentialCoef1,
                   _outputLensDistortionTangentialCoef2,
+                  isCalibrated, totalAvgErr,
                   cameraMatrix, distCoeffs);
-  
-  _outputIsCalibrated->setValue(isCalibrated);
 }
 
-
 bool LensCalibrationPlugin::isIdentity(const OFX::IsIdentityArguments &args, OFX::Clip * &identityClip, double &identityTime)
 {
   return false;
 }
 
 void LensCalibrationPlugin::changedClip(const OFX::InstanceChangedArgs &args, const std::string &clipName)
 {
-  
+  if(args.reason != OFX::InstanceChangeReason::eChangeTime)
+  {
```

Comment:
Maybe nice to add a popup to ask if the user wants to clear all the data or not.
For example, if you are adjusting the contrast on the input clip, it will generate a `changedClip` but you don't want to invalidate all the successfull checkerboard extractions.